### PR TITLE
Some stuff about trading and villages

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/server/world/ServerWorld.mapping [hunk 1]
```diff
@@ -14,7 +14,7 @@ CLASS ve net/minecraft/server/world/ServerWorld
 	FIELD a ticking Z
 	FIELD b savingDisabled Z
 	FIELD c raidManager Laws;
-	FIELD d villageManager Lapu;
+	FIELD d siegeManager Lapu;
```
Comment (Prospector) [inline mappings/net/minecraft/server/world/ServerWorld.mapping]: siege?

Comment (Prospector) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]: oh nvm

**Review by modmuss50 — APPROVED** [review]

**Review by Prospector — APPROVED** [review]

File: mappings/net/minecraft/client/world/ClientChunkManager.mapping [hunk 2]
```diff
@@ -24,6 +24,7 @@ CLASS dgo net/minecraft/client/world/ClientChunkManager
 	FIELD g playerChunkZ I
 	FIELD h world Ldgt;
 	METHOD a updateChunkList ()V
+	METHOD a updateLoadDistance (I)V
```
Comment (Runemoro) [inline mappings/net/minecraft/client/world/ClientChunkManager.mapping:27]: Why not `updateChunkLoadDistance`, to match the packet name?

File: mappings/net/minecraft/entity/mob/ZombieEntity.mapping [hunk 3]
```diff
@@ -1,25 +1,26 @@
 CLASS aus net/minecraft/entity/mob/ZombieEntity
 	CLASS aus$a DestroyEggGoal
-	FIELD bB TICKS_UNTIL_DROWNED Lqg;
+	FIELD bB CONVERTING_IN_WATER Lqg;
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/mob/ZombieEntity.mapping:3]: This is an int, not boolean, so it should be `TICKS_UNTIL_CONVERSION_IN_WATER`.

Comment (Prospector) [reply mappings/net/minecraft/entity/mob/ZombieEntity.mapping:3]: Is it a count up or a count down?

**Comment by modmuss50** [comment]
Sorry about merging without looking at your comments, it came though right as I clicked merge.

File: mappings/net/minecraft/client/world/ClientChunkManager.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/client/world/ClientChunkManager.mapping:27]: It is actually updating the chunk load distance to `arg1 + 2`, and I figured that this is already a chunk manager, so I will just say load distance.

File: mappings/net/minecraft/entity/mob/ZombieEntity.mapping (hunk 3 above)
Comment (liach) [reply mappings/net/minecraft/entity/mob/ZombieEntity.mapping:3]: No, this is a boolean in a data tracker parameter. The remapper failed to recognize this field change.

Comment (Runemoro) [reply mappings/net/minecraft/entity/mob/ZombieEntity.mapping:3]: Though is the `_IN_WATER` suffix necessary?

**Comment by liach** [comment]
For those concerns from @Runemoro, I will explain and make another pr to fix.

File: mappings/net/minecraft/entity/mob/ZombieVillagerEntity.mapping [hunk 4]
```diff
@@ -11,4 +11,5 @@ CLASS aut net/minecraft/entity/mob/ZombieVillagerEntity
 	METHOD a setConverting (Ljava/util/UUID;I)V
 	METHOD a finishConversion (Lve;)V
 	METHOD dX isConverting ()Z
+	METHOD ea getConversionRate ()I
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/mob/ZombieVillagerEntity.mapping:14]: What's "conversion rate"? Do you mean conversion *speed* or conversion *progress*?

Comment (liach) [reply mappings/net/minecraft/entity/mob/ZombieVillagerEntity.mapping:14]: No. It is a rate for one tick, that in that tick, at 1/100 chance, the conversion will check for beds and iron bars around and return a number higher than one to reduce the ticks it take to convert the zombie villager.

File: mappings/net/minecraft/server/world/ServerWorld.mapping [hunk 5]
```diff
@@ -45,6 +45,9 @@ CLASS ve net/minecraft/server/world/ServerWorld
 	METHOD a tickChunk (Lbww;I)V
 		ARG 1 chunk
 		ARG 2 randomTickSpeed
+	METHOD a isInterestingPointAt (Lev;I)Z
```
Comment (Runemoro) [inline mappings/net/minecraft/server/world/ServerWorld.mapping:48]: Are interesting points the same as points of interest? If yes, I think point of interest is better, to be consistent with the other names.

File: mappings/net/minecraft/entity/mob/ZombieEntity.mapping (hunk 3 above)
Comment (liach) [reply mappings/net/minecraft/entity/mob/ZombieEntity.mapping:3]: Yes. The setter call is in a `isIn(FluidTags.WATER)` check.

File: mappings/net/minecraft/server/world/ServerWorld.mapping (hunk 5 above)
Comment (liach) [reply mappings/net/minecraft/server/world/ServerWorld.mapping:48]: There is another integer parameter in the end which indicates the interest level (for a village), with 1 being the village center and 5 being no village. The interest level is associated with chunk sections. The `isVillageAt(BlockPos)` literally calls `isInterestingPointAt(BlockPos, 1)`

**Pull request merged**